.github/workflows/pythonapp.yml

```diff
@@ -35,7 +35,7 @@ jobs:
     - name: Setup Github Cache
       uses: actions/cache@v3
       with:
-        key: shared-cache
+        key: dot-cache-${{ matrix.os }}-${{ matrix.python-version }}-${{ matrix.django }}
```

Comment:
huh? Accidentally commited code? Doesn't fit with the rest of changes

Reply:
No. See above:

> Additional fixes:
> ...
> Split github actions PyPi cache, see: https://github.com/python-poetry/poetry/issues/4568